package.json

```diff
@@ -6,10 +6,10 @@
     "check-types": "tsc --noEmit",
     "start": "node node_modules/react-native/local-cli/cli.js start",
     "test": "npm run format && npm run lint && npm run test:unit",
-    "test:unit": "jest",
+    "test:unit": "jest --passWithNoTests",
     "format": "prettier --write --list-different './src/**/*.{js,ts,tsx}'",
     "lint": "eslint --ext '.js,.ts,.tsx' --fix src && npm run check-types",
-    "precommit": "lint-staged",
+    "precommit": "npm run test",
```

Comment:
I think this should be replaced by `yarn run test`, you should also change other commands to use yarn (except `npm login`)